examples/fixture/asm/kimchi/public_output_bool.asm

```diff
@@ -0,0 +1,29 @@
+@ noname.0.7.0
+
+DoubleGeneric<1>
+DoubleGeneric<1>
+DoubleGeneric<1,1,-1>
+DoubleGeneric<1,0,0,0,-2>
+DoubleGeneric<1,0,-1,0,6>
+DoubleGeneric<1,0,0,0,-8>
+DoubleGeneric<1,1>
+DoubleGeneric<1,1,-1>
+DoubleGeneric<0,0,-1,1>
+DoubleGeneric<1,1>
+DoubleGeneric<1,0,-1,0,1>
+DoubleGeneric<1,-1>
+DoubleGeneric<0,0,-1,1>
+DoubleGeneric<1>
+DoubleGeneric<1,-1>
+(0,0) -> (14,0)
+(1,0) -> (2,1)
```

Comment:
I'm weirded out by the fact that the public output `(1, 0)` is wired to the right wire of the addition gate here